### PR TITLE
Run tests on GitHub Actions

### PR DESCRIPTION
**Pull request by walro**

### CODE REVIEW

**Comment by walro** [comment]
This looks like the first time we will actually have slower execution times on Actions compared to Travis :)

There's some caching that could be done, both for extensions and the composer stuff, but it wasn't as super straight-forward as for other languages with "official" support so I decided not to bother for now. We don't pay for the execution time here anyway.

**Comment by walro** [comment]
> This looks like the first time we will actually have slower execution times on Actions compared to Travis :)

Or well, that is if Travis and GitHub starts at the same time, but Travis always have huge queues so...

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP ${{ matrix.php }}
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          tools: composer:v2
+          extensions: soap
```
Comment (walro) [inline .github/workflows/ci.yml:27]: It's the version of https://github.com/php-vcr/php-vcr that we use that needs `soap`, btw

Comment (walro) [reply .github/workflows/ci.yml:27]: More on that subject, `soap` is pre-installed for 7.1, but not for 5.6 and 7.0, see https://github.com/shivammathur/setup-php/wiki/Php-extensions-loaded-on-ubuntu-18.04

**Comment by walro** [comment]
> There's some caching that could be done, both for extensions and the composer stuff, but it wasn't as super straight-forward as for other languages with "official" support so I decided not to bother for now.

Ah, I found https://github.com/actions/cache/blob/9c77c9dbfc97cd17e1add865174d4622362960f2/examples.md#php---composer so perhaps we can at least do that part as it's "official" enough.

**Comment by walro** [comment]
> Ah, I found https://github.com/actions/cache/blob/9c77c9dbfc97cd17e1add865174d4622362960f2/examples.md#php---composer so perhaps we can at least do that part as it's "official" enough.

Hehe, getting tired. I *did* find this before, but got puzzled by the fact that we didn't have a `.lock` file here, so I skipped it.

**Review by roback — APPROVED** [review]

**Pull request merged**